config/routes.rb

```diff
@@ -142,10 +142,11 @@
       get   "/settings" => "communities#settings",        as: :settings
       patch "/settings" => "communities#update_settings", as: :update_settings
 
+      get "getting_started_guide(/*all)" => "getting_started_guide#index", as: :getting_started_guide
```

Comment:
To my understanding that is not enough.

If you give a name to a path, e.g. `as: :getting_started_guide`, Rails will generate helpers such as `getting_started_guide_path` and `getting_started_guide_url`. Calling for example the path helper `getting_started_guide_path` will return a path (i.e. /admin/getting_started_guide) to the named route. Now the problem is, if there are 2 routes with the same name, which path (/admin/getting_started_guide vs /style_guide/getting_started_guide) the helper will return?